docs/changes.md

```diff
@@ -1,6 +1,8 @@
 ## Changes
 Note that links to pull requests prior to public release of the code (4.0) direct to IBM GitHub Enterprise, and will only be accessible to IBM employees.
 
+- `12.8` Multiple Updates:
+    - Support VPC creation or deletion in AWS Cloud ([#598](https://github.com/ibm-mas/ansible-devops/pull/598))
```

Comment:
For future reference, the entry here and the title of the PR should be the same, I have updated the PR title to match this.